.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -1,24 +1,43 @@
 ---
 name: Bug report
-about: File a bug report in OJS, OMP or OPS. Try our support forum linked below if you can't provide reproduction steps and technical specifications.
-title: ''
+about: File a bug report in OJS, OMP or OPS. Try our Software Support forum, linked below, if you cannot provide reproducible steps and technical specifications.
+title: '[Main Feature] | Workflow Section - Page, Description of bug'
 labels: ''
 assignees: ''
 
 ---
 
-**Describe the bug**
-Please tell us what happens, what you expected to happen, and why you think it is a bug in the software.
+**Description:**
+A detailed explanation of the bug, include any unusual behavior or unique details.  This is the chance to provide some context surrounding the bug.
 
-**To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+**Preconditions or Prerequisites:**
+These are requirements that need to be in place before trying to follow the steps to reproduce bug behavior.  If necessary, it is helpful to include user details, related plugins, submission workflow stage etc... Sometimes the best precondition is that it is seen using "basic install with default dataset"
 
+**Steps to Reproduce:**
+Clear, unambiguous, step-by-step instructions on how to trigger the bug.
+1.  Login with User <role>
```

Comment:
Suggest reducing number of sample steps here so users have less to read through when submitting a bug.